### PR TITLE
imagestreams/httpd-centos: Update for C9S

### PR DESCRIPTION
**Pull request by LorbusChris**
Also switch to using the C8S image from quay.io for EL8 instead of pulling from docker.io, avoiding rate-limits.

### CODE REVIEW

**Comment by LorbusChris** [comment]
cc @phracek

File: imagestreams/httpd-centos.json [hunk 1]
```diff
@@ -21,14 +22,54 @@
...
+          "kind": "DockerImage",
+          "name": "quay.io/fedora/httpd-24:latest"
         },
-        "name": "latest"
+        "referencePolicy": {
+          "type": "Local"
+        }
       },
       {
+        "name": "2.4-micro-fc",
```
Comment (LorbusChris) [inline imagestreams/httpd-centos.json:52]: Fedora images aren't versioned by Fedora release, so I added the `fc` suffix here for fedora-continuous

Comment (phracek) [reply imagestreams/httpd-centos.json:52]: @LorbusChris For CentOS Stream we use c9s or c8s. Are you sure we want to add suffix. C9S and C8S image leaves in different namespace.

Comment (phracek) [reply imagestreams/httpd-centos.json:52]: I see, the point. we can not have two images named `2.4-micro`.

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
Only one important note. We do not test imagestreams against OpenShift 4 environment for Fedora, C8S and C9S. It is not planned right now.

**Pull request merged**